ibc/types.go

```diff
@@ -46,6 +51,27 @@ func (c ChainConfig) Clone() ChainConfig {
 	return x
 }
 
+func (c ChainConfig) VerifyCoinType() (string, error) {
+	// If coin-type is left blank in the ChainConfig,
+	// the Cosmos SDK default of 118 is used.
+	if c.CoinType == "" {
+		typ := reflect.TypeOf(c)
+		f, _ := typ.FieldByName("CoinType")
+		coinType := f.Tag.Get("default")
```

Comment:
This feels odd in the sense that I would expect the struct's field to take on the default value if the empty string is passed in. Idk that I've personally used the `default` tag before so maybe this is the idiomatic way to go about this?

Thoughts? cc @DavidNix @agouin

Reply:
Yes, I thought so too! After testing, if the value is empty, it takes on "0" which is Bitcoin's coin-type.
If there is a better way to go about this, I'm happy to learn.

Reply:
Seems very counterintuitive. I'll go ahead and approve, if there is a better way to do this we can always circle back up on it. Good job!